arduino_sketch/arduino_sketch.ino

```diff
@@ -114,7 +114,7 @@ void loop()
   if (!isSleeping()){
     if (isResting)
     {
-      animateRandomly();
+      //animateRandomly();
```

Comment:
Re-enable to allow random animations. The override of the animations should take place when the pi sends commands, but this has not been tested yet.